src/jobs/sanitization.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require('mongoose');
+const _ = require('lodash');
+const database = require('../utils/database');
+const logger = require('../logger');
+const { sanitizeHtml: { sanitizeDeep } } = require('../utils');
+
+const collectionNames2Paths = new Map([
+	['lessons', 'lessons'], ['news', 'news'], ['homeworks', 'homework'], ['submissions', 'submissions']]);
+const getPath = (collectionName) => collectionNames2Paths.get(collectionName);
+
+
+const collectionName2SafeAttributes = new Map([
+	['helpdocument', ['title', 'content']], ['helpdocuments', ['title', 'content']]]);
+const getSafeAttributes = (collectionName) => collectionName2SafeAttributes.get(collectionName);
+
+const getDiffProps = (a, b) => _.reduce(a, (result, value, key) => (_.isEqual(value, b[key]) ?
```

Comment:
Codacy found an issue: ['?' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5763745)